src/renderer/screens/account/TokensList.js

```diff
@@ -109,27 +113,47 @@ class TokensList extends PureComponent<Props> {
       const tokens = listTokensForCryptoCurrency(currency);
       if (tokens && tokens.length > 0) {
         firstToken = tokens[0];
-        url = supportLinkByTokenType[tokens[0].tokenType];
+        url = supportLinkByTokenType[firstToken.tokenType];
```

Comment:
mmh a bit above, you were using `listTokenTypesForCryptoCurrency(currency) ` which if there is a first value is going to be that tokenType.